### PR TITLE
add fixture loader doc

### PR DESCRIPTION
**Pull request by nicolasThal**

### CODE REVIEW

File: en/developer_guide/custom_fixtures_load.rst [hunk 1]
```diff
@@ -0,0 +1,77 @@
+Custom fixtures load
+====================
+
+Context
+-------
+
+Open Orchestra comes with a set of fixtures that will allow you to have a full overview of the project.
+Those fixtures present you some pages, content, media, and much more.
+
+If you want to have juste the base data which would allow you to login on the Back Office and start
```
Comment (itkg-rboessel) [inline en/developer_guide/custom_fixtures_load.rst]: just

File: en/developer_guide/custom_fixtures_load.rst [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+Then you should configure the ``ModelBundle`` to load only the fixtures implementing the ``BarFixtureInterface``.
+
+.. code-block:: yaml
+
+    open_orchestra_model:
+        production_fixtures_interface:
+            - FooBundle\Loader\BarFixtureInterface
+
+If you want to load the base production fixtures and your personnal fixtures, you can add both interfaces in
```
Comment (itkg-rboessel) [inline en/developer_guide/custom_fixtures_load.rst]: personal

File: en/developer_guide/custom_fixtures_load.rst [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+====================
+
+Context
+-------
+
+Open Orchestra comes with a set of fixtures that will allow you to have a full overview of the project.
+Those fixtures present you some pages, content, media, and much more.
+
+If you want to have juste the base data which would allow you to login on the Back Office and start
+creating your first pages, a simple solution would be to load only the fixtures you need.
```
Comment (itkg-rboessel) [inline en/developer_guide/custom_fixtures_load.rst:11]: you can or other to avoid repetition of "simple solution" after

**Comment by nicolasThal** [comment]
@itkg-rboessel doc updated

File: en/developer_guide/custom_fixtures_load.rst [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+Context
+-------
+
+Open Orchestra comes with a set of fixtures that will allow you to have a full overview of the project.
+Those fixtures present you some pages, content, media, and much more.
+
+If you want to have just the base data which would allow you to login on the Back Office and start
+creating your first pages, a simple solution would be to load only the fixtures you need.
+
+This solution would be to load only the fixtures you want in the command :
```
Comment (juchi) [inline en/developer_guide/custom_fixtures_load.rst]: repetition with the previous sentence

File: en/developer_guide/custom_fixtures_load.rst [hunk 5]
```diff
@@ -0,0 +1,77 @@
+Custom fixtures load
+====================
+
+Context
+-------
+
+Open Orchestra comes with a set of fixtures that will allow you to have a full overview of the project.
+Those fixtures present you some pages, content, media, and much more.
```
Comment (juchi) [inline en/developer_guide/custom_fixtures_load.rst]: is "present" the right verb here ? sounds strange

File: en/developer_guide/custom_fixtures_load.rst [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
+If you want to have just the base data which would allow you to login on the Back Office and start
+creating your first pages, a simple solution would be to load only the fixtures you need.
+
+This solution would be to load only the fixtures you want in the command :
+
+.. code-block:: shell
+
+    $ php app/console doctrine:mongo:fixture:load --fixtures="src" --fixtures="vendor/open-orchestra/"
+
+This solution should be avoided because it won't be functional if the fixtures folder is moving.
```
Comment (juchi) [inline en/developer_guide/custom_fixtures_load.rst]: won't => wouldn't

File: en/developer_guide/custom_fixtures_load.rst [hunk 7]
```diff
@@ -0,0 +1,77 @@
...
+Common usage
+~~~~~~~~~~~~
+
+Open Orchestra provides a console command which allows you to load only the fixtures you need.
+
+.. code-block:: shell
+
+    $ php app/console orchestra:mongo:fixtures:load
+
+This command will look through all the project fixtures and load only the one implementing
```
Comment (juchi) [inline en/developer_guide/custom_fixtures_load.rst]: the one => thoses

File: en/developer_guide/custom_fixtures_load.rst [hunk 8]
```diff
@@ -0,0 +1,77 @@
...
+
+    /**
+     * Class LoadStatusData
+     */
+    class LoadStatusData extends AbstractFixture implements OrderedFixtureInterface, OrchestraProductionFixturesInterface {}
+
+Specific usage
+~~~~~~~~~~~~~~
+
+The previous solution allows you to load only the fixtures needed to login on the Back Office.
```
Comment (juchi) [inline en/developer_guide/custom_fixtures_load.rst]: get logged on

**Comment by nicolasThal** [comment]
@juchi doc updated

**Comment by juchi** [comment]
:+1:

**Pull request merged**